web/user_profile/templates/profile-settings.html

```diff
@@ -11,13 +11,19 @@
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
   {% endblock head %}
+  <script>
+    $(document).ready(function () {
+      $.getScript('{% static 'user_profile/js/update-credentials.js' %}');
+      {% block jquery %} {% endblock jquery %}
+    })
+  </script>
```

Comment:
you dont need to override `document ready`. This code defined in the base.html

Reply:
decided to keep the old implementation